### PR TITLE
Fixed #27266 -- Allowed using assertFormError()/assertFormsetError() in admin forms and formsets.

### PR DESCRIPTION
**Pull request by timgraham**
https://code.djangoproject.com/ticket/27266

### CODE REVIEW

File: django/contrib/admin/helpers.py [hunk 1]
```diff
@@ -304,6 +308,14 @@ def inline_formset_data(self):
         })
 
     @property
+    def forms(self):
+        return self.formset.forms
+
+    @property
+    def non_form_errors(self):
+        return self.formset.non_form_errors
```
Comment (diegueus9) [inline django/contrib/admin/helpers.py]: I just wonder if `self.formset.non_form_errors == self.form.non_form_errors`.

Comment (timgraham) [inline django/contrib/admin/helpers.py]: No, I wouldn't expect that. Maybe the diff doesn't make it clear, but this attribute is on `InlineAdminFormSet` not `AdminForm`.

Comment (diegueus9) [inline django/contrib/admin/helpers.py]: You're right, the diff doesn't show up the difference, but If that method is for `InlineAdminFormSet` what about non_form_errors with an AdminForm?

Comment (timgraham) [inline django/contrib/admin/helpers.py]: Thanks, I pushed a fix for that case.

**Pull request merged**